src/Pages/ContentListTable/components/StatusIcon.tsx

```diff
@@ -43,31 +77,50 @@ const StatusIcon = ({ status, error }: Props) => {
       );
     case 'Invalid':
       return (
-        <Tooltip position='top-start' content={error}>
           <Flex alignContent={{ default: 'alignContentCenter' }} direction={{ default: 'row' }}>
             <FlexItem spacer={{ default: 'spacerSm' }}>
               <ExclamationCircleIcon color={red} />
             </FlexItem>
             <FlexItem>
-              <StatusText color='red'>Invalid</StatusText>
+              <Popover
+                aria-label="invalid popover"
+                alertSeverityVariant='danger'
+                headerContent="Invalid"
+                headerIcon={<ExclamationCircleIcon/>}
+                bodyContent={popoverDescription(error, failedIntrospectionsCount, lastIntrospectionTime)}
+                position='left'
+                footerContent={popoverFooter(retryHandler, uuid)}
```

Comment:
Make this a component, a function type return will force un-needed rerenders.
```suggestion
                footerContent={<PopoverComponent  props=whatevs >}
```

In this case though, with such small

Reply:
Same with description above.